fuzzers/047-hclk-ioi-pips/Makefile

```diff
@@ -3,12 +3,12 @@ PIP_TYPE?=hclk_ioi3
 PIPLIST_TCL=$(FUZDIR)/hclk_ioi3_pip_list.tcl
 TODO_RE=".*"
 # FIXME Modify fuzzer to solve PIPs that drive the BUFIO and BUFR from ILOGIC clock or through the IOI tile.
-EXCLUDE_RE=".*\.HCLK_IOI_((RCLK_IMUX[0-3])|(I2IOCLK))"
+EXCLUDE_RE=".*\.HCLK_IOI_((I2IOCLK)|(IDELAYCTRL))"
```

Comment:
I don't think IDELAYCTRL is needed since here we're excluding PIPs with the specified sources and as far as I know there are only PIPs where HCLK_IOI_IDELAYCTRL_REFCLK is a sink.

Reply:
Done.

Reply:
I actually think that it is needed as the todo files will contain the IDELAYCTRL-related pips, therefore the build will fail as they are no longer supported in this fuzzer